dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp

```diff
@@ -394,7 +395,41 @@ createBusBreakerNetwork(const BusBreakerNetworkProperty& properties) {
         .add();
   }
 
-  if (properties.instantiateLccConverter) {
+  if (properties.instantiateVscConverterWithDisconnectedHvdc) {
+    vl1.newVscConverterStation()
+        .setId("MyVscConverter")
+        .setName("MyVscConverter_NAME")
+        .setConnectableBus("MyBus")
+        .setLossFactor(3.0)
+        .setVoltageRegulatorOn(true)
+        .setVoltageSetpoint(1.2)
+        .setReactivePowerSetpoint(-1.5)
+        .add();
```

Comment:
Why duplicating the code and not just adding the bus depending on the property?

Reply:
In TestDataInterface.cpp, each unit test uses a different network by switching one single attribute in the BusBreakerNetworkProperty struct. I thought creating one new attribute for each new unit test I wanted to add would be good because it keeps the logic of the previous code.

Reply:
yes this is ok, but the code to create the hvdc is mostly copy paste, so it would be better to factorize

Reply:
Ok, I defined 3 functions to factorize the code.